src/shared/components/tc-communities/NewsletterSignup/index.jsx

```diff
@@ -13,6 +13,8 @@ import { fireErrorMessage } from 'utils/errors';
 import qs from 'qs';
 import defaultStyle from './style.scss';
 
+const config = window.CONFIG;
+
```

Comment:
This is not correct. This will work client-side only, but this code is under `/shared` folder, thus should work at the server-side as well (for server-side rendering), you should `import config from 'utils/config';`